adobe/serializable.hpp

```diff
@@ -14,13 +14,16 @@
 #include <ostream>
 #include <type_traits>
 
-#include <adobe/typeinfo.hpp>
 #include <adobe/string/to_string.hpp>
+#include <adobe/typeinfo.hpp>
```

Comment:
Is something automatically reordering these?  How does that play with your include order constraints?

Reply:
No, include order constraints - and yes, clang-format alphabetizes the groups.